programs/programs/mo/pe/spm.py

```
@@ -1,5 +1,25 @@
+    Missouri household receives the federal benefit only.
+
+    ``MoStateCodeDependency`` is load-bearing. ``pe_input()`` never sends
+    ``state_code`` on its own, and PE's Lifeline chain branches on it for both the
+    state supplement and the income limit (TX expands to 150% FPG against the
+    federal 135%). Without it PE falls back to its own default state.
+    """
+
+    pe_inputs = [
+        *Lifeline.pe_inputs,
+        dependency.household.MoStateCodeDependency,
+    ]
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)